model/FullText.inc.php

```diff
@@ -239,56 +290,29 @@ public static function getNewerInLibraryByTime($libraryID, $timestamp, $keys=[])
 		$chars = 0;
 		$data = [];
 		
-		$s3Client = Z_Core::$AWS->createS3();
-		
 		while (($chars < $maxChars) && ($row = array_shift($rows)) && !$stop) {
 			$libraryID = $row['libraryID'];
 			$key = $row['key'];
 			
-			try {
-				$start = microtime(true);
-				$result = $s3Client->getObject([
-					'Bucket' => Z_CONFIG::$S3_BUCKET_FULLTEXT,
-					'Key' => $row['libraryID'] . "/" . $row['key']
-				]);
-				StatsD::timing("s3.fulltext.get", (microtime(true) - $start) * 1000);
+			$data[$key] = self::getItemData($libraryID, $key);
```

Comment:
Simplified a bit. getItemData had practically the same code. I can change back it if no need this.